contracts/TokenVault.sol

```diff
@@ -49,12 +49,18 @@ contract TokenVault is Ownable, Recoverable {
   /** How many tokens investors have claimed */
   mapping(address => uint) public claimed;
 
+  /** When was the last claim by an investor **/
+  mapping(address => uint) public lastClaimed;
+
   /** When our claim freeze is over (UNIX timestamp) */
   uint public freezeEndsAt;
 
   /** When this vault was locked (UNIX timestamp) */
   uint public lockedAt;
 
+  /** defining the tap **/
+  uint public tokensPerSecond;
```

Comment:
How about renaming `tokensPerSecond` to `claimableTokensPerSecond` or `claimableAmountPerSecond`? Without the comment in constructor I can't really guess what `tokensPerSecond` would mean